pypesto/objective/amici_util.py

```diff
@@ -234,7 +234,7 @@ def get_error_output(
     n_res = nt * amici_model.nytrue
 
     return {
-        FVAL: np.inf,
+        FVAL: - np.inf,
```

Comment:
on a second look, I think this was actually right, as the calculator is already supposed to return the nllh values (i.e. flipped sign).

Reply:
however, https://github.com/ICB-DCM/pyPESTO/blob/develop/pypesto/objective/amici.py#L292 seems wrong as the `ret[FVAL] = nllh` already...

Reply:
> however, https://github.com/ICB-DCM/pyPESTO/blob/develop/pypesto/objective/amici.py#L292 seems wrong as the `ret[FVAL] = nllh` already...

Right, the sign is flipped again in `call_unprocessed()`.
As this routine does the right job for non-failing simulations, I would guess we want to keep the minus sign there and just flip it in amici_util...

Or are there other opinions? Happy to discuss. However, this one needs to be merged soon, as it makes optimization impossible for quite some models atm...

Reply:
@paulstapor might be good if the error can be reproduced in a unit test
@FFroehlich can you confirm whether there is a wrong sign?

Reply:
I think `call_unprocessed()` has an extraneous minus sign and `get_error_output` is missing one. Haven't used the steadystate guess code in a while, so I wouldn't be sure its currently doing the right thing ...

Reply:
`get_error_output` is supposed to return the nllh value already. The worst possible value would be `+ np.inf`, which it should thus return, I think.

Reply:
you are right.